### PR TITLE
web: Add version number requested in #8021

### LINKED ISSUE #8021: Access to Ruffle version outside Flash file
**Describe the feature**
Right now the only way to view which version of Ruffle a user has installed is by right clicking inside of a Flash window;
image
A way to check in the pop up or the settings page could be useful, especially with users that do not use nightly builds;
image

### PR DESCRIPTION
**Pull request by aidanbabo**
Change the compile time "macros" such as `%VERSION_NUMBER%` to be contained in only one file `build-info.ts`.

Closes #8021

### CODE REVIEW

**Comment by aidanbabo** [comment]
The script that actually processes the macros (`set_version.js`) still processes every file. I am honestly not too sure how it works, but I am willing to try and make it only process `build-info.ts` if it would speed up compile time and someone wants to direct me a little bit.

File: web/packages/extension/assets/css/popup.css [hunk 1]
```diff
@@ -22,6 +22,10 @@ button {
     color: black;
 }
 
+#version-text {
+    text-align: center;
```
Comment (n0samu) [inline web/packages/extension/assets/css/popup.css:26]: Hello and welcome - this looks great! A little thing I noticed here is that your `version-text` element is a `span`, not a block-level element like a `div`, so your `text-align: center` styling has no effect:
image
If you change it to a div, it looks like what you probably intended:
image

Comment (n0samu) [reply web/packages/extension/assets/css/popup.css:26]: Looks good to me! 👍

**Review by n0samu — APPROVED** [review]

**Review by adrian17 — APPROVED** [review]

**Comment by adrian17** [comment]
Thanks!

**Pull request merged**